backends/tfhe-cuda-backend/cuda/include/integer/integer.h

```diff
@@ -48,6 +48,38 @@ typedef struct {
   uint32_t lwe_dimension;
 } CudaRadixCiphertextFFI;
 
+typedef struct {
+  // divisor-related attributes
+  uint64_t const *divisor_has_at_least_one_set;
+  uint64_t const *decomposed_divisor;
+  uint32_t const num_scalars_divisor;
```

Comment:
These first three attributes should remain outside of the struct. When/if we refactor scalar mul they would be in a `CudaScalarMultiplierFFI` struct.